integration-tests/bats/merge.bats

```diff
@@ -180,7 +180,7 @@ teardown() {
     dolt add test1
     dolt commit -m "add pk 0 = 2,2 to test1"
 
-    run dolt merge merge_branch
+    run dolt merge merge_branch --no-commit
```

Comment:
since this test is testing merge failing on a conflict, it might give us a little more coverage to specify a commit message here, instead of `--no-commit` since we expect the merge to fail and not commit anyway. That would just help test the case where the user requested to commit, but merge conflicts prevent it.